tools/clang-func-mapping/ClangFnMapGen.cpp

```diff
@@ -182,19 +130,20 @@ class MapFunctionNamesAction : public ASTFrontendAction {
   }
 };
 
+static cl::extrahelp CommonHelp(CommonOptionsParser::HelpMessage);
+
 int main(int argc, const char **argv) {
   // Print a stack trace if we signal out.
   sys::PrintStackTraceOnErrorSignal(argv[0], false);
   PrettyStackTraceProgram X(argc, argv);
 
   SmallVector<std::string, 4> Sources;
+  const char *overview =
+      "\nThis tool collects the mangled name and location of definition "
+      "of all functions defined in the given compilation command.\n";
```

Comment:
This tool dumps the definitions that available in the C/C++ files (omitting definitions from headers). Not definitions from the compilation commands.